src/main/java/com/lesson45/service/BankingService.java

```diff
@@ -0,0 +1,141 @@
+package com.lesson45.service;
+
+import com.lesson45.dto.TransferCardToCardDTO;
+import com.lesson45.model.Card;
+import com.lesson45.model.Client;
+import com.lesson45.postgresDriverManager.PostgresDriverManager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class BankingService {
+    private PostgresDriverManager postgresDriverManager;
+
+    public Client getUserById(int id) throws SQLException {
+        String sql ="""
+                    SELECT clients.client_id ,card_id, name ,balance, card_number FROM clients
+                    JOIN cards ON clients.client_id = cards.client_id 
+                    WHERE clients.client_id = ?;
+                    """;
+
+        try (Connection connection = postgresDriverManager.getConnection();
+             PreparedStatement preparedStatement = connection.prepareStatement(sql)) {
+            preparedStatement.setInt(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+
+            Client client = null;
+            List<Card> cards = new ArrayList<>();
+            while (resultSet.next()) {
+                if (client == null) {
+                    client = new Client();
+                    client.setId(resultSet.getInt("client_id"));
+                    client.setName(resultSet.getString("name"));
+                }
+
+                Card card = new Card();
+                card.setId(resultSet.getInt("card_id"));
+                card.setCardNumber(resultSet.getString("card_number"));
+                card.setBalance(resultSet.getBigDecimal("balance"));
+                cards.add(card);
+            }
+            if (client != null) {
+                client.setCards(cards);
+                return client;
+            } else {
+                throw new SQLException("Пользователь с id: " + id + " не найден!");
```

Comment:
WE shouldn't throw such exception for get method.
In general, throwing an exception is not expected here